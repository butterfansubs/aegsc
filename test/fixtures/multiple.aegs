Outside 1
%[ template line ^ Comment @ 1 < 0:12:34.567 > 1:23:45.678 $ Template1 # Actor =L 10 =R 20 =V 30
  { %; Comment in override
    \pos(!
      $x
        + 10 -- Add 10
    !,! %; Comment in inline code
      $y
        + 20
    !)
    \c&H000000&
  }
  %; %] Comment containing end marker
  Line: %; Comment outside override
%]

Outside 2
%[ code line ^ Comment @ 10 < 3:45:67.890 > 2:34:56.789 $ Code1 # Actor2 =L 10 =R 20 =V 30
  %; Aegs Comment
  -- Lua comment
  function xModYPlusZ(x, y)
    local z = 5
    return x % y + 5 %; 6
    --[[
      Lua multiline %; Comment
    ]]
  end
%]

Outside 3
%[ code once ^ Comment @ 20 < 4:56:78.999 > 5:67:89.000 $ Code2 # Actor3 =L 11 =R 21 =V 31
  %; Aegs Comment
  -- Lua comment
  function xModYPlusZ2(x, y)
    local z = 5
    return x % y + 15 %; 16
    --[[
      Lua multiline %; Comment
    ]]
  end
%]
%[ template syl ^ Dialogue @ 30 < 6:78:90.123 > 7:89:01.234 $ Template2 # Actor4 =L 12 =R 22 =V 32
  { %; Comment in override
    \pos(!
      $x
        + 20 -- Add 20
    !,! %; Comment in inline code
      $y
        + 20
    !)
    \c&HFFFFFF&
  }
  %; %] Comment containing end marker
  Line: %; Comment outside override
%]
