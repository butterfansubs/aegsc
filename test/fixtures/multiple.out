Comment: 1,0:12:34.567,1:23:45.678,Template1,Actor,10,20,30,template line,{\pos(!$x+10!,!$y+20!)\c&H000000&}Line:
Comment: 10,3:45:67.890,2:34:56.789,Code1,Actor2,10,20,30,code line,function xModYPlusZ(a,b)local c=5;return a%b+5 end
Comment: 20,4:56:78.999,5:67:89.000,Code2,Actor3,11,21,31,code once,function xModYPlusZ2(a,b)local c=5;return a%b+15 end
Dialogue: 30,6:78:90.123,7:89:01.234,Template2,Actor4,12,22,32,template syl,{\pos(!$x+20!,!$y+20!)\c&HFFFFFF&}Line:
